src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeStorageAccountCredential.md

```diff
@@ -25,7 +25,7 @@ The **New-AzDataBoxEdgeStorageAccountCredential** cmdlet creates a new edge stor
 
 ### Example 1
 ```powershell
-New-AzDataBoxEdgeStorageAccountCredential -ResourceGroupName resourceGroupName -DeviceName device-name -Name storage-acount-credential-name -StorageAccountName storageAccountName -StorageAccountType BlobStorage -StorageAccountAccessKey @SecureString -EncryptionKey @SecureString
+New-AzDataBoxEdgeStorageAccountCredential -ResourceGroupName resourceGroupName -DeviceName device-name -Name storage-acount-credential-name -StorageAccountType BlobStorage -StorageAccountAccessKey @SecureString -EncryptionKey @SecureString
 ```
```

Comment:
-Name storage-acount-credential-name -StorageAccountName storageAccountName
↓
-Name storageAccountName
(not sure)